README.md

```diff
@@ -16,6 +16,7 @@ This project bundles all Icinga PHP libraries into one piece and can be integrat
 * [ipl-stdlib](https://github.com/Icinga/ipl-stdlib)
 * [ipl-validator](https://github.com/Icinga/ipl-validator)
 * [ipl-web](https://github.com/Icinga/ipl-web)
+* [ipl-scheduler](https://github.com/Icinga/ipl-scheduler)
```

Comment:
The list is alphabetically sorted, please insert this at the appropriate position.